src/controller/students/createStudentController.js

```diff
@@ -17,10 +18,12 @@ export const createStudentController = async (req, res) => {
 
         const newStundent = await createStudent(Student, body)
         if(newStundent.error){
+            studentLogger.info('Student not created due to incorrect data received');
```

Comment:
It can be useful to log the incorrect data that produced the error (maybe at 'debug' level)